index.html

```diff
@@ -17,36 +18,34 @@
   <script defer src="dist/second-level-navigation.js"></script>
   <script defer src="dist/search.js"></script>
 </head>
+
 <body>
-<!-- <img class="mock" src="/mock.jpg" alt=""> -->
+  <!-- <img class="mock" src="/mock.jpg" alt=""> -->
 
   <div class="page-wrapper">
     <a href="#main-content" class="skip-link visually-hidden focusable">Skip to main content</a>
-  <!-- Header (at wide widths) has height specified so content will not jump -->
+    <!-- Header (at wide widths) has height specified so content will not jump -->
     <header class="site-header">
 
       <!-- Gets fixed by JS at wide widths -->
       <div class="site-header__fixable fixable">
         <div class="header__left">
-          <button class="nav-primary__button" aria-controls="site-header__inner" aria-label="Toggle navigation" aria-expanded="false">
+          <button class="nav-primary__button" aria-controls="site-header__inner" aria-label="Toggle navigation"
+            aria-expanded="false">
             <span class="nav-primary__icon"></span>
           </button>
         </div>
 
         <!-- Needs to extend full width so box shadow will also extend -->
         <div id="site-header__inner" class="site-header__inner">
           <div class="container">
-            <div class="grid-full">
+            <div class="grid-full grid-full-no-js">
```

Comment:
Adding the class by default, and then removing it in the JS has the potential to create a flash of unstyled content -- especially for users on slow connections.

A better way would be making use of the the `js` CSS class that Drupal inserts onto the body field when JavaScript is enabled. 

```css
html:not(.js) .site-header__inner .grid-full { ... }
```

Use that in conjunction with a stylesheet that's added within a `<noscript>` tag, and we won't have any flash of unstyled content.